service-loadbalancer/service_loadbalancer.go

```diff
@@ -413,6 +423,12 @@ func (lbc *loadBalancerController) getServices() (httpSvc []service, tcpSvc []se
 				newSvc.SessionAffinity = true
 			}
 
+			if strings.EqualFold(s.Namespace, api.NamespaceDefault) {
```

Comment:
Do you actually need the service name for anything or can we just call the field `path`? I'd prefer having a single function called getServicePathForLBRule that takes a Service+port and returns the path for it. This may/may-not include namespace port etc based on some if statements. That makes it easier to integrate into Ingress as well (https://github.com/bprashanth/contrib/commit/dca689f7e6f805d54c90717814a86b8f0a696c04#diff-75fc7741fc92747587a186ae42240c0eR427).